hyperspy/misc/ipython_tools.py

```diff
@@ -30,16 +30,17 @@ def get_ipython():
     """
     if is_it_running_from_ipython is False:
         return None
-    import IPython
-    ipy_version = Version(IPython.__version__)
+    from IPython import __version__ as ipythonversion
```

Comment:
Here, `IPython` was imported only for the `version`, because all other uses have additional `import from` statements. Again, it would maybe make sense and remove all the `import from` statements by adding the module name to the calls.